go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/oglok/nimbess-cni
+
+go 1.12
```

Comment:
if using 1.12 is the GO111MODULE still used? Other question, I thought we were on 1.11 currently, so do we want to use 1.12 now?

Reply:
Changed back to 1.11.